docs/hello-neighbour.yaml

```diff
@@ -135,11 +172,11 @@ paths:
       produces:
       - "application/json"
       parameters:
-      - in: "path"
-        name: "churchUuid"
+      - name: "churchUuid"
+        in: "path"
+        required: true
         type: "string"
         format: "uuid"
-        required: true
       - in: "body"
```

Comment:
I think it would be good to allow names and dates of birth here as well (but not make them required) and the possibly allow leaders to lock it down, so people cannot update those.

Reply:
Added a `MemberInvite` object for this